app/routes/docs/newproducers.mdx

```diff
@@ -0,0 +1,70 @@
+---
+handle:
+  breadcrumb: New Producers
+---
+
+import {
+  ProcessList,
+  ProcessListItem,
+  ProcessListHeading,
+  SummaryBox,
+} from '@trussworks/react-uswds'
+import { GitHubLabel } from '~/components/GitHubLabel'
+
+# Setting up New GCN Notice Producers
+
+The following steps guide new instrument, mission, or observatory producers
+into setting up new notices streams that are distributed to the user community
+via [Kafka](faq#what-is-kafka). This process requires interactions
+with the [GCN Team](https://heasarc.gsfc.nasa.gov/cgi-bin/Feedback?selected=kafkagcn)
+to enable these accounts and topics are setup on the GCN Kafka broker,
+and we are happy to work together to help you define schema utilizing our
+core JSON schema.
+
+## Defining Kafka Topics and JSON Schema
+
+The preferred format for formatting of the data sent via GCN Kafka is JSON.
+We have designed a set of [core schema](https://github.com/nasa-gcn/gcn-schema/tree/master/core-schema)
+which serves as the building blocks for your Notices.
+You can also define instrument-specific quantities in your
+schema, but we request that you utilize the core schema as much as possible.
+Please add your schema to the [GCN repository](https://github.com/nasa-gcn/gcn-schema).
+
+The GCN team is happy to iterate with the producers on their schema contents and format.
+Your pipeline will generate JSON files following these schema and send to GCN
+in the following process.
+
+## Steps to Start Publishing Alerts
+
+1. Make sure that all of the producer users (who will enable GCN notices in your pipelines)
+   have signed in (at least once) to https://test.gcn.nasa.gov and https://gcn.nasa.gov.
+2. Please send the [GCN Team](https://heasarc.gsfc.nasa.gov/cgi-bin/Feedback?selected=kafkagcn)
+   a list of email addresses of all the people who should have the
+   producer access to the MissionName.\* topics.
```

Comment:
Italicize _mission_name_ so that it is clear that it is template text:

```suggestion
   producer access to the <code><i>mission_name</i>.*</code> topics.
```